db/db_impl_secondary.h

```diff
@@ -62,11 +64,19 @@ class LogReaderContainer {
   };
 };
 
+// The secondary instance shares access to the storage as the primary.
+// The secondary is able to read and replay changes described in both the
+// MANIFEST and the WAL files without coordination with the primary.
+// The secondary instance can be opened using `DB::OpenAsSecondary`, and can
```

Comment:
I am concerned about the ambiguity of the last sentence because there is no way for the secondary instance to keep up-to-date with the primary if the primary has a lot of writes. Calling `TryCatchUpWithPrimary` can cause the secondary to continue to replay the logs, but there is no guarantee for the secondary to catch up with the primary.

Reply:
how about rephrase with "and can make best effort attempt to catch up with the primary using  `DBImplSecondary::TryCatchUpWithPrimary`."

Reply:
SG.